src/AppContext.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react'
+
+export type AppContextType = {
```

Comment:
New Context for the SageAccountWeb app, to make the appId and redirectURL available to any sub-components in the tree